src/Tools/ExternalAccess/OmniSharp/ImplementType/OmniSharpImplementTypeOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Internal;
+using Microsoft.CodeAnalysis.ImplementType;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.ImplementType
+{
+    internal static class OmniSharpImplementTypeOptions
+    {
+        public static PerLanguageOption<OmniSharpImplementTypeInsertionBehavior> InsertionBehavior =
+            new PerLanguageOptionInternalConverter<OmniSharpImplementTypeInsertionBehavior, ImplementTypeInsertionBehavior>(ImplementTypeOptions.InsertionBehavior);
```

Comment:
❔ Don't we have an `AsPublicOption` extension or something similar?

Reply:
Since `ImplementTypeInsertionBehavior` isn't public, I would actually recommend using helper methods to get/set the option, and not actually exposing the option directly.